server/views/pages/appointments/create-and-edit/bulk-appointments/upload-bulk-appointment.njk

```diff
@@ -18,17 +19,32 @@
                     id: "file",
                     name: "file",
                     label: {
-                        text: "Upload a CSV file to create appointments",
+                        text: "Upload your list of prison numbers",
                         classes: "govuk-label--l",
                         isPageHeading: true
                     },
+
+                    hint: {
+                        text: "Your list of prison numbers must be in a CSV file. You can include appointment start
+                        and end times using the 24-hour clock. For example 14:00 not 2:00pm"
+                    },
                     errorMessage: validationErrors | findError('file')
                 }) }}
-
+                {{ govukDetails({
+                  summaryText: "How to use a CSV file",
+                  text: howToUseCSVFile()
+                }) }}
                 {{ govukButton({
                     text: "Continue"
                 }) }}
             </form>
         </div>
     </div>
 {% endblock %}
+
+{% macro howToUseCSVFile() %}
+   <p class="govuk-body">A CSV file is a type of spreadsheet. Download the template if you need one,</p>
+   <p class="govuk-body">then add the relevant prison numbers. When you upload it to this service a</p>
+   <p class="govuk-body">list of attendees will be created.</p>
+   <p class="govuk-body"> Download the <a href="/assets/downloads/prisoner-list.csv" download>prison number list template</a> [CSV, 1kb]</p>
```

Comment:
There are actually two example csv files. The one for bulk appointments is /assets/downloads/bulk-appointments.csv

Reply:
added civ for bulk-appointments